iroh-sync/src/store.rs

```diff
@@ -175,6 +181,23 @@ pub trait Store: std::fmt::Debug + Clone + Send + Sync + 'static {
 
     /// Get peers to use for syncing a document.
     fn get_sync_peers(&self, namespace: &NamespaceId) -> Result<Option<Self::PeersIter<'_>>>;
+
+    /// Set the download policy for a document.
+    fn set_download_policy(&self, namespace: &NamespaceId, policy: DownloadPolicy) -> Result<()>;
```

Comment:
why is this not on `DownloadPolicyStore`?

Reply:
I thought about it as a read only approach. It's intentional but yeah could be ready and write